change/@ni-nimble-components-2ad496e4-6c7a-4418-bd07-1c716e45e196.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@fredvisser any idea why Chromatic isn't showing a diff for the new icons? They appear in the Storybook build. If I inspect the chromatic build there are a bunch of icons missing from the snapshot, but no diff reported.

https://www.chromatic.com/component?appId=60e89457a987cf003efc0a5b&csfId=icons&buildNumber=5680&k=645d1ba0515c6e7e87c5a16f-1200-snapshot-true-firefox&h=50&b=-13